src/components/home/NetworkCard.tsx

```diff
@@ -28,6 +39,10 @@ const NetworkCard: React.FC<{ network: Network }> = ({ network }) => {
       extra={<StatusBadge status={network.status} />}
       onClick={handleClick}
     >
+      <Row>
+        <Styled.title>{`Description`}</Styled.title>
+        <Styled.description>{network.description}</Styled.description>
+      </Row>
```

Comment:
It doesn't look good for networks without a description set. I also don't like the height of the card varying based on how long the description is. I would suggest using an "info" icon with a tooltip next to the name, and only display it if a description is defined. 
![image](https://github.com/jamaljsr/polar/assets/1356600/98775e5a-963e-4006-b5f2-ea752f79c17b)